tests/test_helper/lxd.bash

```diff
@@ -1,10 +1,9 @@
 function launch_containers() {
-    local image_name=$1; shift
```

Comment:
I see no reason to remove this argument, we might need tests targeting explicit versions?

Reply:
I wanted to cut down on moving parts. I was thinking about possible scenarios, and came to the conclusion that the choice of the image version would likely be global for all tests in the suite, rather than a configurable option that every test sets individually.
That being said, I'm not dead set on removing it.

Reply:
I guess that's fair, less is more, we can add it back if we actually need individual control.